functions/src/modules/SE.functions.ts

```diff
@@ -77,3 +79,94 @@ export const processNewAllotment = functions.database
     });
     return 1;
 });
+
+
+
+
+/////////////////////////////////////////////////
+//
+//          UploadProcessing (Storage Trigger)
+//      1. Send a notification to the coordinator
+//      2. Update the Task
+//
+/////////////////////////////////////////////////
+
+export const uploadProcessing = functions.storage.object()
+.onFinalize(async object => {    
+
+    const filePath = object.name;
+
+
+    if (!filePath.startsWith('sound-editing/restored/uploads'))
+        return -1;
+    
+    
+    const filePathRegex = /sound-editing\/restored\/uploads\/(\w+)\/(\w+)\/(\w+\/)*(([\w\d]+)-\d+-?\d+?)\.flac/;
+    
+    const match = filePathRegex.exec(filePath);
+    if (!match) {
+        console.log("Wrong Path -- File will be deleted.");
```

Comment:
Here and everywhere add a file path to the log message. Unless it is somehow specified automatically.